src/components/paginator/paginator.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { LeftArrow, RightArrow, Wrapper, Text, TextWrapper } from './styles';
+
+interface PaginatorProps {
+  onRightClick: () => void;
+  onLeftClick: () => void;
+  currentPage: number;
+  lastPage: number;
+  isLastPage: boolean;
+}
+
+const Paginator = ({ onRightClick, onLeftClick, currentPage, lastPage, isLastPage }: PaginatorProps): JSX.Element => {
```

Comment:
Sugiro tipar componentes React com `React.FC`. Assim não precisa tipar o retorno da função, e a tipagem fica mais específica e mais clara tbm.

```suggestion
const Paginator: React.FC<PaginatorProps> = ({ onRightClick, onLeftClick, currentPage, lastPage, isLastPage }: PaginatorProps) => {
```